Companies page

isaacfreeman opened the pull request:
Here's an initial PR for a companies page. What improvements do we need?

Commit:
Companies page

malclocke commented on index.html:
```diff
@@ -54,6 +54,8 @@
             </li>
           </ul>
 
+          <h3><a href="companies">Ruby Companies in New Zealand</a></h3>
```
should be `companies.html` I think.

malclocke replied:
The link looks a bit weird in the `h3` text.  I think it might look better as:

```
<h3>Ruby Companies in New Zealand</h3>

<p>Looking for help with a Ruby website in New Zealand?  Here is a link with some <a href="companies.html">pithy text</a></p>
```

Commit:
Tidied up index page

parndt commented:
Maybe we should save blurbs for the companies' websites?

jimsynz commented:
This seems more than a little problematic. We have an agenda item for our next meeting to allow paid corporate members - once we do that perhaps we can use that as a source of truth for this page?

isaacfreeman commented:
It's useful for readers to know what companies specialise in, and allowing them to write a blurb is a simple way to do that.

Paid members might be allowed to add a little badge, perhaps.

isaacfreeman commented:
@jamesotron So long as the blurbs are in first person, I think it's clear that responsibility for accuracy lies with each company, not with Ruby NZ. I'm not aware of that ever being a problem for drupal.org.nz.

jimsynz commented:
seems legit. :+1:

isaacfreeman commented:
Right-o. There's lots more that could be done to improve the presentation, but at this point I'll invite other people on the thread to add their own companies, and we'll see how much interest there is. I'm happy to do some more work on CSS and layout and such, but only if it looks like the page will be used.

mhaylock commented:
Restricting companies to either a very short blurb, or simply a list of specialties might be better. Right now a single company occupies a very large portion of the page. I wouldn't want to see a page like this become a battle for attention.

Randomised ordering within each region might be a good thing to do also.

isaacfreeman commented:
@mhaylock has rightly pointed out that it's going to be a bit hard for people to contribute when the PR is coming from Resolve's fork directly to gh-pages.

How about we close this PR and start a new branch for the companies page? I'll send a PR to that branch with an empty page and another with a listing for Resolve. Then other people can make their own PRs for their company info, and we can merge it into gh-pages when we're happy that it has enough listings to give a reasonable overview of the NZ Ruby community. Plan?

isaacfreeman commented:
I do agree on keeping the blurbs short, and the one I've done seems too long. Perhaps we could make it a single short paragraph plus up to five bullet points about your specialities.

parndt commented:
We've elected to go with a more simple list of companies. Information can include:
- Company name
- Logo (I suppose included with the PR and we're allowed to display it?)
- Location

In future we may expand this, but for now that's what we're willing to display. cc @nzruby/committee

mhaylock commented:
Excellent :+1:

isaacfreeman commented:
Cool.

Commit:
Updates from PR discussion

- removes blurb
- adds other companies from 2010 page

isaacfreeman commented:
Updated with the simple list, and companies from the 2010 page.

Based on some quick Google searches, I've removed those that don't seem to be in business anymore, and renamed those that seem to have new names. I haven't included logos for companies apart from Resolve.

megahbite commented:
Seems to have some individual contractors in there. Should probably restrict it to listed companies?

danielfone commented:
> Based on some quick Google searches, I've removed those that don't seem to be in business anymore, and renamed those that seem to have new names. I haven't included logos for companies apart from Resolve.

@isaacfreeman wow good work! This is an excellent starting point.

> Should probably restrict it to listed companies?

@megahbite what's your thinking behind this? Do we want to have criteria? If so, are there other criteria we should think of?

megahbite commented:
@danielfone Seems a bit odd to have individuals mixed in with companies is all.

parndt commented:
Individual contractors should be allowed here, I reckon!

megahbite commented:
Then let's :shipit:

isaacfreeman commented:
I think individual contractors should definitely be included, as they're relevant for what I think is the primary audience: clients with a Ruby project who need a developer.

However, there might be a secondary audience of developers looking for employment. For their purposes it might be useful to show a distinction between individual contractors and organisations that employ. Doesn't have to be in the initial version, though, and it might be obvious enough in practice.

parndt commented:
Of course independent contractors can also employ, or subcontract :)

charlespeach commented:
@isaacfreeman We have a new design for the website now. Wondering if you would like to look at potentially updating this to fit the new site, or would you prefer we close this off?

Tis a shame this never got merged as its good stuff 👍

nickmalcolm commented:
As per recent Slack discussion around helping folk find jobs, and just general awareness of the Ruby landscape in NZ, I'm ➕ 1ing this PR! I might have a go at reviving it if I find time.... 🔎

libbyschuknight commented:
@nickmalcolm I am happy to try and help with this!

isaacfreeman commented:
Thanks @nickmalcolm, @libbyschuknight. Fix it up however you like.

libbyschuknight commented:
Suggestion from @aimeelegge on Ruby NZ Slack:
>I'd like to add my employer. Is it worth having an option to link to current openings on the list?
